src/parser/jobs/smn/modules/Ruin2.js

```diff
@@ -115,42 +102,23 @@ export default class Ruin2 extends Module {
 	}
 
 	_onComplete() {
-		const potLossPerR2 = RUIN3_POT - RUIN2_POT
-		// It would be a waste to not use any stacks of Further Ruin left after the
-		// last weave of the fight, so do not warn about them.
-		const issues = this._issues.filter(cast => { return this.isNotEndOfFightRuin4(cast) }).length
-		const warnings = this._warnings.filter(cast => { return this.isNotEndOfFightRuin4(cast) }).length
-
-		this.suggestions.add(new TieredSuggestion({
-			icon: ACTIONS.RUIN_III.icon,
-			tiers: BAD_CAST_SEVERITY,
-			value: issues,
-			content: <Trans id="smn.ruin-ii.suggestions.issues.content">
-				Prioritise casting <ActionLink {...ACTIONS.RUIN_III}/>. Casting <ActionLink {...ACTIONS.SMN_RUIN_II}/> or <ActionLink {...ACTIONS.RUIN_IV}/> is a DPS loss when not used to weave oGCDs or proc <ActionLink {...ACTIONS.WYRMWAVE}/>s.
-				Unnecessary Ruin 4 casts will require an extra Ruin 2 cast for a future oGCD weave.
-			</Trans>,
-			why: <Trans id="smn.ruin-ii.suggestions.issues.why">
-				{issues * potLossPerR2} potency lost to {issues} unnecessary Ruin II or Ruin IV
-				<Plural value={issues} one="cast" other="casts"/>.
-			</Trans>,
-		}))
+		const wastedCount = this._worthless.length
+		const moveOnlyCount = this._moveOnly.length
+		const weavedCount = this._weaved.length
+		const totalCount = wastedCount + moveOnlyCount + weavedCount
 
 		this.suggestions.add(new TieredSuggestion({
 			icon: ACTIONS.SMN_RUIN_II.icon,
-			content: <Trans id="smn.ruin-ii.suggestions.warnings.content">
-				Unless significant movement is required, avoid using <ActionLink {...ACTIONS.SMN_RUIN_II}/> or <ActionLink {...ACTIONS.RUIN_IV}/> only for movement. Most position adjustments can be performed with slidecasting and the additional mobility available during <ActionLink {...ACTIONS.DREADWYRM_TRANCE}/>.
+			tiers: BAD_CAST_SEVERITY,
+			value: totalCount,
+			content: <Trans id="smn.ruin-ii.suggestions.avoid.content">
+				Avoid casting <ActionLink {...ACTIONS.SMN_RUIN_II}/> if possible.
+				Prefer using your other instant casts for movement and weaving or Ruin III when movement and weaving are not required.
 			</Trans>,
-			why: <Trans id="smn.ruin-ii.suggestions.warnings.why">
-				{warnings * potLossPerR2} potency lost to {warnings} Ruin II or Ruin IV
-				<Plural value={warnings} one="cast" other="casts"/>
-				used only to move.
+			why: <Trans id="smn.ruin-ii.suggestions.avoid.why">
+				{totalCount} Ruin II <Plural value={totalCount} one="cast" other="casts"/> were performed.
+				({weavedCount} for weaving or triggering Wyrmwave, {moveOnlyCount} for only movement, and {wastedCount} for no benefit.)
```

Comment:
Not particularly on-board with this, it's an incredibly long `why`. If it can't be simplified, it's probably a sign that the single suggestion is relying on too many individual details.

In this case, I'd err towards just simplifying.

Reply:
I've shortened this to just the total count.  The other counts were there to give extra data to let the user know "how bad" the ruin 2 casts were.  I had considered outputting it as a separate table below, but it doesn't feel like enough data for that.